paper-slider.html

```diff
@@ -474,7 +474,7 @@
       ready: function() {
         // issue polymer/polymer#1305
         this.async(function() {
-          this._updateKnob(this.value);
+          this._updateKnob(this.value, this.min, this.max);
         }, 1);
```

Comment:
@frankiefu demo looks good and tests still pass. Should I remove it (and hopefully nothing breaks)?